src/components/Gallery.js

```diff
@@ -1,13 +1,40 @@
 import React from 'react'
+import {connect} from 'react-redux';
 
-export default function Gallery() {
+import { increaseCounter, decreaseCounter} from "../redux/Counter/counter.actions"
+
+// The counter and the counter buttons is just to check that 
+// The redux setup is working this can be deleted.
+function Gallery(props) {
     return (
         <div className="gallery">
-             <span role="img" aria-label="books-emoji">📚📚</span>
-                     BOOKS GALLERY 
-                <span role="img" aria-label="books-emoji">📚📚</span>
+            <span role="img" aria-label="books-emoji">📚📚</span>
+                BOOKS GALLERY 
+            <span role="img" aria-label="books-emoji">📚📚</span>
+            <div>Count: {props.count}</div>
+
+            <button onClick={() => props.increaseCounter()}>Increase Count</button>
+
+            <button onClick={() => props.decreaseCounter()}>Decrease Count</button>
         </div>
-    )
+    );
 }
 
+const mapStateToProps = state => {
+    console.log(state);
```

Comment:
Remove this `log` statement

Reply:
Hello, 

Alright I would make those changes. 

As for the redux folder structure, the issue said to setup redux for the project and so I created a folder for all things redux. 

In there i created a file that handles the creation of this redux store that is then imported in index.js file

I also created a file named rootReducer which would hold all the reducers that the application might be using

Then finally I created a folder for an example feature that will be dealing with state for the application, the reason for doing this is that it isnt stated yet what states that the application would be handling yet. So any component in our react application that would be handling state would have a corresponding redux folder and inside the redux folder it would have type, actions and reducers to handle the state for that component.

Hopefully this explains it well

Reply:
Yes, this makes sense. We will work with your arrangement, thanks!